proto/cosmos/auth/beta/auth.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package cosmos.auth;
+package cosmos.auth.beta;
```

Comment:
about this: an alternate folder arch would be `cosmos.beta.auth`, `cosmos.v1.base`? etc. wdyt?

Reply:
Actually no, let's follow Buf's best practices, and put the version at the end: https://buf.build/docs/lint-checkers/#package_version_suffix.